internal/core/monitoring/metrics.go

```diff
@@ -0,0 +1,95 @@
+package monitoring
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/topfreegames/maestro/internal/config"
+	"go.uber.org/zap"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+type MetricOpts struct {
+	Namespace  string
+	Subsystem  string
+	Name       string
+	Help       string
+	Labels     []string
+	Buckets    []float64
+	Objectives map[float64]float64
```

Comment:
Do you have anything in mind for this attribute? Right now, it is not being used.

Reply:
Removing